Libraries/LibGUI/GFilePicker.cpp

```diff
@@ -95,12 +117,17 @@ GFilePicker::GFilePicker(const StringView& path, CObject* parent)
     filename_label->set_size_policy(SizePolicy::Fixed, SizePolicy::Fill);
     filename_label->set_preferred_size({ 60, 0 });
     auto* filename_textbox = new GTextBox(filename_container);
+    if (m_mode == Mode::Save) {
+        filename_textbox->set_text("Untitled.txt");
```

Comment:
`.txt` seems completely specific to the text editor application.
A more general solution would be to let `GFilePicker` have a "preferred extension" or something like that.
You don't have to implement that now, just something to think about for the future!